modules/core/src/test/java/org/locationtech/jts/operation/relateng/RelatePointLocatorTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2023 Martin Davis.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v20.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+package org.locationtech.jts.operation.relateng;
+
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.Location;
+
+import junit.textui.TestRunner;
+import test.jts.GeometryTestCase;
+
+public class RelatePointLocatorTest extends GeometryTestCase {
+  
+  public static void main(String args[]) {
+    TestRunner.run(RelatePointLocatorTest.class);
+  }
+  
+  public RelatePointLocatorTest(String name) {
+    super(name);
+  }
+  
+  String gcPLA = "GEOMETRYCOLLECTION (POINT (1 1), POINT (2 1), LINESTRING (3 1, 3 9), LINESTRING (4 1, 5 4, 7 1, 4 1), LINESTRING (12 12, 14 14), POLYGON ((6 5, 6 9, 9 9, 9 5, 6 5)), POLYGON ((10 10, 10 16, 16 16, 16 10, 10 10)), POLYGON ((11 11, 11 17, 17 17, 17 11, 11 11)), POLYGON ((12 12, 12 16, 16 16, 16 12, 12 12)))";
+  
+  public void testPoint() {
+    //String wkt = "GEOMETRYCOLLECTION (POINT(0 0), POINT(1 1))";
+    checkLocation(gcPLA, 1, 1, DimensionLocation.POINT_INTERIOR);
+    checkLocation(gcPLA, 0, 1, Location.EXTERIOR);
```

Comment:
I don't thing this test comparing a Location to a DimensionLocation makes sense

Reply:
In fact `DimensionLocation.EXTERIOR = Location.EXTERIOR` (see [here](https://github.com/locationtech/jts/blob/master/modules/core/src/main/java/org/locationtech/jts/operation/relateng/DimensionLocation.java#L26)).  So this is techically correct - but it would be better to use the `DimensionLocation` value.   I'll fix that.

Also, would be better if the check method was called `checkDimLoc` to be clear - I'll change that too.

Reply:
Fixed: https://github.com/locationtech/jts/commit/8c71ae7b56f600ed05b87c9584ae04fe33d154dd